src/lsst/cst/__init__.py

```diff
@@ -1,14 +1,18 @@
 """Rubin LSST Community Science Tools"""
 
-__all__ = ["__version__"]
-
 from importlib.metadata import PackageNotFoundError, version
 
+from .mpl_utils import remove_figure
+
 __version__: str
 """The version string of lsst
 (PEP 440 / SemVer compatible).
 """
 
+
+__all__ = ["__version__", "remove_figure"]
```

Comment:
I think `__all__` is usually a tuple for immutability.

Reply:
Except all the Python docs use a list for their examples...

Reply:
Like I said, it's an optional suggestion 🙂